Library/Homebrew/os/mac/xcode.rb

```diff
@@ -28,8 +28,8 @@ def latest_version
       end
 
       def prerelease?
-        # TODO: bump to version >= "8.1" after Xcode 8.0 is stable.
-        version > "7.3.1"
+        # TODO: bump to version >= "8.2" after Xcode 8.1 is stable.
+        version > "8.1"
```

Comment:
Should this be `>= "8.1"` or `> "8.0"` instead of `> 8.1`? 8.0 is the stable version now.